pynumdiff/linear_model/_linear_model.py

```diff
@@ -1,115 +1,25 @@
-import copy, math, logging, scipy
+import math, scipy
 import numpy as np
 from warnings import warn
 
-from pynumdiff.finite_difference import first_order as finite_difference
+from pynumdiff.finite_difference import second_order as finite_difference
```

Comment:
A last case where the former-first-order-which-was-actually-second-order was still being imported as first order, which is now properly first order. I haven't been checking on lineardiff as closely, so this slipped through.